qiskit/tools/backends/reduced_cmap.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2019, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""Reduced coupling_map
+"""
+
+import numpy as np
+import scipy.sparse as sp
+import scipy.sparse.csgraph as cs
+from qiskit.qiskiterror import QiskitError
+
+
+def reduced_coupling_map(backend, mapping):
```

Comment:
Why not a static method in the `CouplingMap` class?

Reply:
The whole point of this routine is to grab a subgraph of a device so it is more explicit to have the backend actually there as an arg.